broskichat-app/src/Components/Partials/Header.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+// import logo from '../../logo.svg'; // Use logo later
+
+class Header extends React.Component {
```

Comment:
@akeon-lee lets try to be consistent with our file formatting. We should use the .jsx extension and 2 spaces instead of 4.